lib/src/ui/pages/home/news/primary_news/primary_news_list_entity.dart

```diff
@@ -18,7 +18,7 @@ class PrimaryNewsListEntity {
   });
 }
 
-final primaryNewsListEntities = <PrimaryNewsListEntity>[
+final List<PrimaryNewsListEntity> primaryNewsListEntities = <PrimaryNewsListEntity>[
```

Comment:
w jakich sytuacjach decydujemy się ustawiać typ, a w jakich nie?

Reply:
Generalnie użyłem domyślnych zasad linta, tutaj naruszało `type_annotate_public_apis`:
https://dart-lang.github.io/linter/lints/type_annotate_public_apis.html

Reply:
pewnie dlatego, że to top level zmienna

Reply:
dokładnie